src/testdir/Make_dos.mak

```
@@ -31,6 +31,12 @@ report:
 		else ( echo ALL DONE )
 
 clean:
+	-del *.out
+	-del *.failed
+	-del *.res
+	-if exist $(DOSTMP) rd /s /q $(DOSTMP)
```

```suggestion
	-if exist dostmp rd /s /q dostmp
```
`DOSTMP` is not defined now.